ldap/servers/slapd/conntable.c

```diff
@@ -181,13 +181,13 @@ connection_table_new(int table_size)
             ct->c[ct_list][i].c_fdi = SLAPD_INVALID_SOCKET_INDEX;
 
             if (pthread_mutex_init(&(ct->c[ct_list][i].c_mutex), &monitor_attr) != 0) {
-                slapi_log_err(SLAPI_LOG_ERR, "connection_table_get_connection", "pthread_mutex_init failed\n");
+                slapi_log_err(SLAPI_LOG_ERR, "connection_table_new", "pthread_mutex_init failed\n");
                 exit(1);
             }
 
             ct->c[ct_list][i].c_pdumutex = PR_NewLock();
             if (ct->c[ct_list][i].c_pdumutex == NULL) {
-                slapi_log_err(SLAPI_LOG_ERR, "connection_table_get_connection", "PR_NewLock failed\n");
+                slapi_log_err(SLAPI_LOG_ERR, "connection_table_new", "PR_NewLock failed\n");
                 exit(1);
             }
 
```

Comment:
I think we shouldn't just "yolo commit" when obvious faults exist, because we'll forgetn about the issue and never fix it, then in a years time we'll be like "oh fuck why did this break".

So IMO fix it now. If we identify a problem, resolve it.

Reply:
It could be me that says that, I agree with ye guys, the right thing is to fix it now. Will do. Thanks